src/components/FavoriteDrinksGrid/FavoriteDrinksGrid.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import DrinkDetails from '../DrinkDetails/DrinkDetails'
+import NavBar from '../NavBar/NavBar'
+
+function RecipeGrid() {
+  return (
+    <div>
```

Comment:
I'm noticing a lot of `divs` as your outer-most wrappers in your JSX. Have you looked into using [fragments](https://react.dev/reference/react/Fragment)? Or are there more semantic HTML elements to reach for?